services/sidecar/rollup/rpc/bridge/bridge_client.go

```diff
@@ -50,3 +51,49 @@ func (c *BridgeClient) GetLastConfirmedAssertionID(ctx context.Context) (*big.In
 func (c *BridgeClient) GetRequiredStakeAmount(ctx context.Context) (*big.Int, error) {
 	return c.IRollup.CurrentRequiredStake(&bind.CallOpts{Pending: false, Context: ctx})
 }
+
+func (c *BridgeClient) RequireFirstUnresolvedAssertionIsRejectable(ctx context.Context, address common.Address) error {
+	return c.IRollup.RequireFirstUnresolvedAssertionIsRejectable(&bind.CallOpts{Pending: false, Context: ctx}, address)
+}
+
+func (c *BridgeClient) IsStakedOnAssertion(ctx context.Context, assertionID *big.Int, address common.Address) (bool, error) {
+	return c.IRollup.IsStakedOnAssertion(&bind.CallOpts{Pending: false, Context: ctx}, assertionID, address)
+}
+
+// Returns the last assertion ID that was validated *by us*.
```

Comment:
any reason to keep these commented out functions?

Reply:
they were there before, I just moved the code. we can clean that up @ujvl I think you're listed on git blame, is it fine to remove?

Reply:
yes remove

Reply:
done